network-api/networkapi/tests/test_middleware.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import MagicMock
+
+from django.test import TestCase
+
+from networkapi.utility.middleware import ReferrerMiddleware, XRobotsTagMiddleware
```

Comment:
Considering that the classes under test are located in the `utility` app, have you considered moving the tests into that app as well (i.e. `network-api/networkapi/utility/tests/test_middleware`)?

Reply:
Updated!